src/app/public/modules/colorpicker/colorpicker.component.html

```diff
@@ -278,6 +283,10 @@
       'sky-colorpicker-reset-button': 'default',
       'sky-btn sky-btn-icon-borderless sky-colorpicker-reset-button-modern': 'modern'
     }"
+    [disabled]="disabled"
```

Comment:
Code style > check your alphabetical ordering here.

Reply:
done in last commit